client/src/app/App.scss

```
@@ -18,6 +18,17 @@
         @include desktop-down {
             margin-top: calc(2 * #{$navbar-height-mobile});
         }
+
+        .disclaimer-banner {
+            margin: 30px;
+            padding: 20px;
+            background-color: $iota2-teal-500;
```

just a suggestion to prevent blindness :sweat_smile: 
```suggestion
            padding: 20px;
            border-radius: 0;
            border: 1px solid $iota2-teal-500;
            background-color: rgba($iota2-teal-500, 0.2);
```

---

Done in [6716589](https://github.com/iotaledger/explorer/pull/1496/commits/671658991975218949f0d4d915c9cdd364752d2c)